IBPSA/Fluid/HeatPumps/ModularReversible/RefrigerantCycle/BaseClasses/PartialCarnot.mo

```diff
@@ -1,12 +1,14 @@
 within IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.BaseClasses;
 partial model PartialCarnot
   "Model with components for carnot efficiency calculation"
-
+  parameter Boolean useForChi "=false to use in heat pump models";
   parameter Boolean useAirForCon
```

Comment:
Do these two parameters (`useAirForCon` and `useAirForEva`) only affect assumptions on default values? If so and if we only need to know if the medium is air or water, please refer to how the fan/pump model does this by looking at the default medium density, so that no manual input is needed.

https://github.com/ibpsa/modelica-ibpsa/blob/cf6d6cdd5347bf490728f33405e76be5a67f1c20/IBPSA/Fluid/Movers/FlowControlled_dp.mo#L53-L59

Reply:
Yes, but the medium is not in this model. However, I changed to Boolean to just `cpEva` and `cpCon`.